src/layout/section/textFieldsSection/TextFieldsSection.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { space } from '../../../_utils/branding'
+import { TextFieldProps } from '../../../textField'
+import { BaseSection, BaseSectionProps } from '../baseSection'
+
+export type TextFieldsSectionProps = BaseSectionProps &
+  Readonly<{
+    children: React.ReactElement<TextFieldProps>[]
+    className?: string
+    tag?: JSX.Element
+  }>
+
+/**
+ * Inputs Section: display two or more TextFieldProps on a row.
+ */
+const TextFieldsSectionDefinition = (props: TextFieldsSectionProps) => {
+  const { className, children, tag = <div /> } = props
+
+  return (
+    <BaseSection
+      {...props}
+      tagName={tag.type}
```

Comment:
It looks like  `BaseSection` is expecting a string, maybe it would be to keep it that way in this component too.
`tagName : string` and  tagName={tagName} for consistency. WDYT?

Reply:
Agree 🙂 Updating that